src/Akeneo/Connectivity/Connection/front/tests/src/webhook/components/EventLogBadge.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import {EventLogBadge} from '@src/webhook/components/EventLogBadge';
+import {render, screen} from 'akeneo-design-system/lib/storybook/test-util';
```

Comment:
You should not depend on `akeneo-design-system` internals.

```suggestion
import {render, screen} from '@testing-library/react';
```
^ not sure about the import

Reply:
It's a failed auto import, I never intended to depends on the dsm for our tests.
Good catch !